### PR TITLE
David

### PR DESCRIPTION
**Pull request by dadkofrajer**

### CODE REVIEW

**Comment by MIWdlB** [comment]
Please leave a discription about what you want to pull in. Is this related to an issue?

**Comment by dadkofrajer** [comment]
I want to add in a string to binary function

File: Communication_test.py [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
 from machine import Pin
 import time
 
 red = Pin(15, Pin.OUT)
 green = Pin(14, Pin.OUT)
 
+
+def string_to_binary(x):
+  l = [(ord(i)) for i in x]
+  m = [(int(bin(i)[2:])) for i in l]
```
Comment (MIWdlB) [inline Communication_test.py]: What does this do?

```suggestion
  m = [bin(i)[2:] for i in l]
  m = [int(item) for item in m]
```

Comment (MIWdlB) [reply Communication_test.py]: Leave a comment explaining why we need to slice [2:]

**Review by MIWdlB — CHANGES_REQUESTED** [review]
Make the list comprehension easy for me to read, and add a comment explaining why we need slices.

File: Communication_test.py (hunk 1 above)
Comment (dadkofrajer) [reply Communication_test.py]: @MIWdlB The comment is now there

**Review by DrOrnage — APPROVED** [review]

**Pull request merged**